spring-data-neo4j/src/test/java/org/springframework/data/neo4j/repository/support/SimpleNeo4jRepositoryTests.java

```diff
@@ -0,0 +1,91 @@
+package org.springframework.data.neo4j.repository.support;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+import org.neo4j.ogm.cypher.query.Pagination;
+import org.neo4j.ogm.cypher.query.SortOrder;
+import org.neo4j.ogm.session.Session;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+
+public class SimpleNeo4jRepositoryTests {
+
+	private final Session sessionMock = mock(Session.class);
+	private final PageRequest twoElementPageRequest = PageRequest.of(0, 2);
+	private final PageRequest oneElementPageRequest = PageRequest.of(0, 1);
```

Comment:
I'd like to see a test in the middle of something, like 5 elements in total, requesting 2nd page with pagesize 2 as well.